libs/accounts/email-renderer/src/templates/verifyLoginCode/index.ts

```diff
@@ -0,0 +1,42 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+export type TemplateData = {
+  code: string;
+  passwordChangeLink: string;
+  serviceName: string;
+  time: string;
+  device: {
+    uaBrowser: string;
+    uaOSVersion: string;
+    uaOS: string;
+  };
+  location: {
+    stateCode: string;
+    country: string;
+    city: string;
+  };
+  date: string;
+
+  cmsRpClientId?: string;
+  cmsRpFromName?: string;
+  entrypoint?: string;
+  subject?: string;
```

Comment:
Is `subject` not always required?

Reply:
These are all cms attributes, so they are optional by default. See strapi.stories.ts vs index.stories.ts.